enterprise-suite/console-api/prometheus.yml

```diff
@@ -84,9 +84,10 @@ scrape_configs:
         regex: (.*)-[0-9]{13}-(exec|driver)-[0-9]+;
         replacement: ${1}-${2}
         target_label: es_workload
-    # flink workload — of the form 'taxi-ride-fare-seglo-processor-dde77c7d-jm' or 'taxi-ride-fare-seglo-processor-dde77c7d-tm'
-      - source_labels: [kubernetes_pod_name, es_workload]
-        regex: (.*)-[a-z,0-9]{8}-(jm|tm);
+      # flink workload — of the form 'taxi-ride-fare-seglo-processor-dde77c7d-jm' or 'taxi-ride-fare-seglo-processor-dde77c7d-tm'
+      - source_labels: [kubernetes_pod_name]
+        action: replace
+        regex: '(.*)-[a-z,0-9]{8}-(jm|tm)-[456789bcdf]{4,10}-[bcdfghjklmnpqrstvwxz2456789]{5}'
```

Comment:
Are you sure this is right? It doesn't seem to match any of the examples prior.

Reply:
I've deleted the example.